user_interface.py

```diff
@@ -51,23 +57,42 @@ def write_products(file):
         product = catalog.BreadGoods(name, price)
         catalog.products.append(product)
 
-print("Do you wish to write the list into a .txt file")
+
+print("Do you wish to write the list into a database?")
 answer = input("type Y or N ").lower()
 if answer == "y":
-    print("Do you wish to use manual directory input? If not then script's directory will be used")
-    answer = input("type Y or N ").lower()
-    if answer == "y":
-        file_directory = input("Please redeem the directory of the file you are going to use (example D:\list\goods) ")
-        file = f'{file_directory}.txt'
-        write_products(file)
-    elif answer == "n":
-        file_directory = os.getcwd()
-        file_name = input("Please redeem your file's name ")
-        file = f'{file_directory}\{file_name}.txt'
-        write_products(file)
-    else:
-        print("You have to print either Y or N")
+    try:
+        # connect to exist database
+        connection = psycopg2.connect(
+            host=host,
+            user=user,
+            password=input("Please redeem your database password (You can change other parameters such as database name, user and host inside the config file) "),
+            database=db_name
+        )
+
+        connection.autocommit = True
+
+        try:
+            with connection.cursor() as cursor:
+                 cursor.execute(
+                     """CREATE TABLE item_catalog(
+                         code INT PRIMARY KEY,
```

Comment:
This code is unique, so cannot ne created bu random integer.

Reply:
Is doing it this way fits the requirements?